rules/xcodeproj.bzl

```diff
@@ -913,6 +915,21 @@ https://www.rubydoc.info/github/CocoaPods/Xcodeproj/Xcodeproj/Constants
         "additional_bazel_build_options": attr.string_list(default = [], mandatory = False),
         "bazel_execution_log_enabled": attr.bool(default = False, mandatory = False),
         "bazel_profile_enabled": attr.bool(default = False, mandatory = False),
+        "lldb_log_file": attr.string(default = "", mandatory = False, doc = """
```

Comment:
As a user I often have some custom LLDB init code I'd like to run - what do you think about just exposing an API to enable that. In XCHammer generated project this works because I can just add an entry to `$HOME/.lldbinit` - to customize the lldb init for debugging or other features. That might already be possible today in rules_ios.

That seemed beneficial because it didn't change the project generated but still allows debugging: as a user I can just pop a line into the lldbinit!

Reply:
[We create one LLDB configuration file per target](https://github.com/bazel-ios/rules_ios/blob/master/rules/xcodeproj.bzl#L507)

But I guess there's value in making this PR a way to set multiple additional LLDB settings instead of "log enable" only?

If we would follow the path of using `$HOME/.lldbinit` no changes are required in `rules_ios` right? Since Xcode will pick it up. The goal of this PR was to abstract editing scripts and/or LLDB configuration files directly to simplify the user experience.

I think making the changes here generic for any list of LLDB settings instead of "log enable" only is a good middle-ground between:

1. Assuming you know what you're doing, edit `$HOME/.lldbinit` and let Xcode do its thing
2. Let `rules_ios` talk to LLDB by just setting `xcodeproj` attributes for "log enable" only

I'd update this PR to set any list of LLDB settings passed in in all `.lldbinit` files for each target in that case via an `xcodeproj` attribute.

cc @jerrymarino thoughts?